### PR TITLE
Feat (admin process list) init admin process list

### PR DESCRIPTION
**Pull request by VishalThapaliya**
Covers [BPO-428](https://bonitasoft.atlassian.net/browse/BPO-428)
Covers [BPO-430](https://bonitasoft.atlassian.net/browse/BPO-430)
Covers [BPO-431](https://bonitasoft.atlassian.net/browse/BPO-431)
Covers [BPO-434](https://bonitasoft.atlassian.net/browse/BPO-434)
Covers [BPO-435](https://bonitasoft.atlassian.net/browse/BPO-435)
Covers [BPO-437](https://bonitasoft.atlassian.net/browse/BPO-437)
Covers [BPO-440](https://bonitasoft.atlassian.net/browse/BPO-440)

### CODE REVIEW

File: uid-pages/page-admin-process-list/src/AdminProcessListBonitaV1.json [hunk 1]
```diff
@@ -0,0 +1,4219 @@
...
+                                            "propertyValues" : {
+                                              "cssClasses" : {
+                                                "type" : "constant",
+                                                "value" : "item-label"
+                                              },
+                                              "hidden" : {
+                                                "type" : "constant",
+                                                "value" : false
+                                              },
+
```
Comment (abirembaut) [inline uid-pages/page-admin-process-list/src/AdminProcessListBonitaV1.json]: I guess the comment should be changed (still talking about cases)

Comment (abirembaut) [inline uid-pages/page-admin-process-list/src/AdminProcessListBonitaV1.json]: same comment

File: uid-pages/page-admin-process-list/src/resources/page.properties [hunk 2]
```diff
@@ -0,0 +1,5 @@
+displayName=Bonita Admin Process List
+name=custompage_AdminProcessListBonita
+description=Page allowing to list all the processes for admin profile
+resources=[GET|bpm/process]
```
Comment (abirembaut) [inline uid-pages/page-admin-process-list/src/resources/page.properties]: I guess some resources need to be added (like the PUT for the state and the get for the user)

Comment (DumitruCorini) [reply uid-pages/page-admin-process-list/src/resources/page.properties]: There is no GET for the user, it is a deploy, but for the PUT, I'll add it

File: uid-pages/page-admin-process-list/src/AdminProcessListBonitaV1.json (hunk 1 above)
Comment (DumitruCorini) [reply uid-pages/page-admin-process-list/src/AdminProcessListBonitaV1.json]: fixed

Comment (DumitruCorini) [reply uid-pages/page-admin-process-list/src/AdminProcessListBonitaV1.json]: fixed

File: uid-pages/page-admin-process-list/src/AdminProcessListBonitaV1.json [hunk 3]
```diff
@@ -3989,7 +3989,7 @@
         "    $data.enabledProcessData.enabledProcessesForNextPage = $data.enabledProcessUrl.slice($data.enabledProcessData.numberOfEnabledProcessesPerPage);",
         "    // saving the other half to be displayed later",
         "    $data.enabledProcessData.numberOfEnabledProcessesReturned = $data.enabledProcessData.enabledProcessesForNextPage.length;",
-        "    // set open cases to undefined as the script is triggered when click on load more cases button",
+        "    // set enabled processes to undefined as the script is triggered when click on load more cases button",
         "    // and to prevent this block of code from being executed (prevent resetting the displayedOpenCases)",
```
Comment (abirembaut) [inline uid-pages/page-admin-process-list/src/AdminProcessListBonitaV1.json]: displayedOpenCases should be changed

**Comment by DumitruCorini** [comment]
For the refresh, it is already done

File: uid-pages/page-admin-process-list/src/AdminProcessListBonitaV1.json [hunk 4]
```diff
@@ -4132,7 +4132,7 @@
         "    $data.disabledProcessData.disabledProcessesForNextPage = $data.disabledProcessUrl.slice($data.disabledProcessData.numberOfDisabledProcessesPerPage);\r",
         "    // saving the other half to be displayed later\r",
         "    $data.disabledProcessData.numberOfDisabledProcessesReturned = $data.disabledProcessData.disabledProcessesForNextPage.length;\r",
-        "    // set open cases to undefined as the script is triggered when click on load more cases button\r",
+        "    // set disabled processes to undefined as the script is triggered when click on load more cases button\r",
         "    // and to prevent this block of code from being executed (prevent resetting the displayedOpenCases)\r",
```
Comment (abirembaut) [inline uid-pages/page-admin-process-list/src/AdminProcessListBonitaV1.json]: displayedOpenCases should be changed

**Pull request merged**